corehq/apps/app_manager/models.py

```diff
@@ -4878,6 +4881,16 @@ def wrap(cls, data):
 
         return self
 
+    # combines title_label values and translation_title_label values into one dict
+    def _get_label_dict(self, data, default_dict, translated_dict):
```

Comment:
Using [update](https://python-reference.readthedocs.io/en/latest/docs/dict/update.html) would simplify this, likely to the point where you wouldn't need a helper function.